Library/Homebrew/test/gpg_spec.rb

```diff
@@ -13,7 +13,12 @@
         shutup do
           subject.create_test_key(dir)
         end
-        expect(dir/".gnupg/secring.gpg").to exist
+
+        begin
+          expect(dir/".gnupg/pubring.kbx").to be_file
+        rescue RSpec::Expectations::ExpectationNotMetError
+          expect(dir/".gnupg/secring.gpg").to be_file
+        end
```

Comment:
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/gpg.rb#L22 should help